### ISSUE TITLE
Problem with Article::allTags();

### ISSUE THREAD

**Issue by JackieDo** [rtconner/laravel-tagging#29]
When i use Article::allTags(); in my ArticlesController, I got QueryException (42S22):

```sql
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'count' in 'field list' (SQL: select distinct `tag_slug`, `tag_name`, `count` from `tagging_tagged` where `taggable_type` = Article order by `tag_slug` asc)
```

I lookat my table "tagging_tagged", I don't see any field with name "count". Why I have this problem?

**Comment by rtconner** [comment]
Dag, how did my tests miss this. Ouch. Ok thanks.

**Comment by rtconner** [comment]
Fixed on tag 0.5.1
